src/app/services/storage.service.ts

```diff
@@ -167,4 +167,13 @@ export class BrowserStorageService {
   set singlePageAccess(val) {
     this.set('singlePageAccess', val);
   }
+
+  get stackConfig() {
+    const result = this.get('stackConfig');
+    return result || false;
```

Comment:
should not be `false` by default, it should be an object, so default is `null` or `{}`, and that is covered in the `this.get()` already, so don't need to specify it here.